packages/components/MentoringSection.tsx

```diff
@@ -57,7 +57,7 @@ function CardItem({ title, description }: CardItemProps) {
 }
 
 const mentoringList = [
-  'self-guided-mentorships',
+  'study-mentorships',
```

Comment:
i thought that "self guided" was a nice fit , but if @frattezi agrees on this  change ,it's fine for me .

Reply:
I liked it too but, thinking about it, self-guided and mentorship are kinda unrelated. The expression is also not known.